pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -26,6 +26,8 @@ type (
 		SharedNodeSecurityGroup string `json:"sharedNodeSecurityGroup,omitempty"`
 		// +optional
 		AutoAllocateIPv6 *bool `json:"autoAllocateIPv6,omitempty"`
+		// +optional
+		NATGateway string `json:"natGateway,omitempty"`
```

Comment:
When you make any changes in this package, you should run `make generate-kubernetes-types` and add generated output.